tests/test_transforms/test_inference/conftest.py

```diff
@@ -28,6 +28,72 @@ def regular_ts(random_seed) -> TSDataset:
     return tsds
 
 
+@pytest.fixture
+def regular_ts_one_month(random_seed) -> TSDataset:
```

Comment:
I don't really think we should check all this cases in `test_inference`. Thy aren't that different. Let's only keep e.g. only this fixture.